lib/circular_buffer.ex

```diff
@@ -69,6 +84,17 @@ defmodule CircularBuffer do
 
   @doc """
   Checks the buffer to see if its empty
+
+  Returns `true` if the given circular buffer is empty, otherwise `false`.
```

Comment:
This function wasn't covered via `test/circular_buffer_test.exs`. So, I took the chance to add some docs here.